devicetypes/smartthings/zigbee-motion-sensor-light.src/led-cpx-light.groovy

```diff
@@ -0,0 +1,166 @@
+/**
+ *  Copyright 2022 SmartThings
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License. You may obtain a copy of the License at:
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *  for the specific language governing permissions and limitations under the License.
+ *
+ *  LED CPX light
+ *
+ *  Author: SAMSUNG LED
+ *  Date: 2022-01-05
+ */
+
+import physicalgraph.zigbee.zcl.DataType
+import groovy.json.JsonOutput
+
+metadata {
+	definition(name: "LED CPX light", namespace: "SAMSUNG LED", author: "SAMSUNG LED", runLocally: true, minHubCoreVersion: '000.019.00012', executeCommandsLocally: true, genericHandler: "Zigbee") {
```

Comment:
Those flags after `author` really shouldn't be here. If you're witnessing errors in IDE, they're caused by something else.

Reply:
I think that is characteristic of parent-child device. If I deleted those, device works very confused. I attached logs when I send on/off control after deleted.

3:37:37: debug description is catchall: 0104 0006 01 01 0000 00 FC19 00 00 0000 0B 01 0B00
3:37:37: debug description is catchall: 0104 0006 01 01 0000 00 FC19 00 00 0000 0B 01 0100
3:37:37: debug description is on/off: 1
3:37:31: debug description is catchall: 0104 0006 01 01 0000 00 FC19 00 00 0000 0B 01 0B00
3:37:30: debug description is catchall: 0104 0006 01 01 0000 00 FC19 00 00 0000 0B 01 0000
3:37:30: debug description is on/off: 0

From bottom to top 3 lines are "off" and 3 lines are "on". The location of log is just before line @73. They are came in to the parser. The parser send each of them as an event. Then plug-in works incorrectly "off" "off" "off" "on" "on" "off" but device works correctly "off" "on".

Reply:
Can you comment out line 70 and see what happens?